elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemSchema.java

```diff
@@ -54,8 +54,9 @@ public enum ElytronSubsystemSchema implements PersistentSubsystemSchema<ElytronS
     VERSION_17_0(17),
     VERSION_18_0(18),
     VERSION_18_0_COMMUNITY(18, Stability.COMMUNITY),
+    VERSION_19_0_PREVIEW(19, Stability.PREVIEW),
```

Comment:
There is no reason to increment the numeric version.
The next schema version would just be 18.0:preview.